src/west/_bootstrap/main.py

```diff
@@ -222,7 +222,7 @@ def bootstrap(args):
           os.path.join(directory, WEST_DIR, MANIFEST))
 
     # Parse the manifest and look for a section named "west"
-    manifest_file = os.path.join(directory, WEST_DIR, MANIFEST, 'default.yml')
+    manifest_file = os.path.join(directory, WEST_DIR, MANIFEST, 'manifest.yml')
```

Comment:
`west-manifest.yml` is also fine with me.
Just wanted something different that `default.yml`

Reply:
second thought `west-manifest.yml` could imply describing `west` (which it can in optional `west` section), but it is describing projects.
And someone could decide to use something else / manuel parsing the file for fetching dependencies.

How about : `projects.yml` ?
@carlescufi any thought ?

Reply:
I do not like `west-manifest.yml` for the reasons outlined by @tejlmand and because it's too long, and I dislike `projects.yml` because it's as generic as `manifest.yml`, so we are not solving the collision issue. To be honest I would maybe call it `west.yml`. That avoids collisions and, if we ever decide to stop using the word `manifest` then the files will not need to be renamed.

Reply:
@carlescufi a little puzzled on your comment, maybe you can clarify.
First you wrote:
> I do not like west-manifest.yml for the reasons outlined by @tejlmand 

but I was addressing the `west` part in the name `west-manifest.yml` when I wrote:
> west-manifest.yml could imply describing west (which it can in optional west section), but the file is describing projects.
> And someone could decide to use something else / manuel parsing the file for fetching dependencies.

so how does the suggestion `west.yml` address those comments ?

Reply:
Guess my main point is that:
`projects.yml` just lists all projects in use / required for the `self`-project to work.

Then user can use:
- `west` to fetch and parse `projects.yml` to stay up-to-date with latest Zephyr
- Read `projects.yml`manually, and use those to fork / copy /  whatever into his own system, to base his own development on Zephyr. But in this example, he may have no intention of keeping up-to-date with Zephyr development. Kind of what was asked by @aescolar on: How to use Zephyr without `west`
- make his own script / tool to fetch sub-projects

So `projects.yml` has no expectations on how / which tool you use, it just describes projects.
Anything having `west` in its name, starts to imply that `west` is required.

Reply:
>@carlescufi a little puzzled on your comment, maybe you can clarify.

`west-manifest.yml` might imply "this is the manifest for west itself", but `west.yml` for me is like `CMakeLists.txt` or `Makefile`: a file that is processed by west.

The issue with `projects.yml` is that it doesn't solve the problem we set out to solve: collisions. If we are going to ignore the fact that a name can collide with existing files, then I say let's use `manifest.yml`, because we are using the term "manifest repository" extensively.

Reply:
To summarize our brief conversation with @tejlmand:

- If we dismiss the possibility of ~collusion~ collision with additional files, then `projects.yml` might be a very descriptive name
- If we instead admit that there is a definite risk of ~collusion~ collision, then my personal prefence is `west.yml` because it is consistent with how similar tools operate and with the rest of the files that west creates or uses (`.west/`, `west`, `west.yml`)
- @tejlmand has proposed `west-projects.yml` but I personally think this is too long, especially if we allow nested manifests further down the line

Reply:
NO COLLUSION! WITCH HUNT! ;)

More seriously, I like your proposal of `west.yml`, @carlescufi.

Reply:
@carlescufi technically you made the `west-projects.yml` in the chat but probably based upon  some of my other comments, and then I just had a little preference on that name compared to `west.yml`

But let's go with `west.yml`

Reply:
@tejlmand yes sorry, I did propose it myself, you are right.